### PR TITLE
Feature 366/add criticality

### PR DESCRIPTION
**Pull request by opoliarush**
Added criticality to insights: 
 - Many Queries
- Chatty API
- bottleneck

image
image
image

### CODE REVIEW

File: src/components/Insights/ExcessiveAPICallsInsight/styles.ts [hunk 1]
```diff
@@ -30,3 +30,12 @@ export const SpanName = styled.span`
...
 `;
+
+export const RowStat = styled.div`
+  display: flex;
+  font-size: 14px;
+  flex-direction: row;
+  gap: 8px;
+  overflow: hidden;
+  padding-top: 12px;
+`;
```
Comment (kshmidt) [inline src/components/Insights/ExcessiveAPICallsInsight/styles.ts:41]: Seems unused
```suggestion
```

File: src/components/Insights/HighNumberOfQueriesInsight/styles.ts [hunk 2]
```diff
@@ -37,3 +37,7 @@ export const Key = styled.span`
 export const IconContainer = styled.span`
   display: flex;
 `;
+
+export const RowStat = styled(Stat)`
+  flex-direction: row;
+`;
```
Comment (kshmidt) [inline src/components/Insights/HighNumberOfQueriesInsight/styles.ts:43]: Seems unused

```suggestion
```

File: src/components/Insights/InsightCard/styles.ts [hunk 3]
```diff
@@ -102,3 +102,11 @@ export const AsyncBadge = styled.div`
...
   }};
 `;
+
+export const RowStat = styled.div`
+  display: flex;
+  font-size: 14px;
+  flex-direction: row;
+  gap: 8px;
+  overflow: hidden;
+`;
```
Comment (kshmidt) [inline src/components/Insights/InsightCard/styles.ts:112]: Seems unused

```suggestion
```

File: src/components/Insights/SpanBottleneckInsight/styles.ts [hunk 4]
```diff
@@ -33,3 +33,17 @@ export const ButtonsContainer = styled.div`
...
   gap: 8px;
 `;
+
+export const RowStat = styled.div`
+  display: flex;
+  font-size: 14px;
+  flex-direction: row;
+  gap: 8px;
+  overflow: hidden;
+`;
```
Comment (kshmidt) [inline src/components/Insights/SpanBottleneckInsight/styles.ts:43]: Seems unused

```suggestion
```

**Review by kshmidt — CHANGES_REQUESTED** [review]

**Review by kshmidt — APPROVED** [review]

**Review by kshmidt-digma — DISMISSED** [review]

**Review by kshmidt-digma — APPROVED** [review]

**Pull request merged**